client/src/AppBundle/Controller/UserController.php

```diff
@@ -19,9 +22,49 @@
 use Symfony\Component\Security\Core\Authentication\Token\UsernamePasswordToken;
 use Symfony\Component\Security\Core\Exception\UsernameNotFoundException;
 use Symfony\Component\Translation\TranslatorInterface;
+use TheSeer\fDOM\CSS\Translator;
```

Comment:
Is this the right Translator we need for this class?

Reply:
Nope 😆 it's not even used the `TranslatorInterface` in the line above is the one used in the class :D removed 👍🏻